pkgs/by-name/dm/dmitry/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
@dotlambda we can remove both of them since they are not used.